docs/TestCoverage.md

```diff
@@ -9760,7 +9760,6 @@ data = np.array([[[
 ]]], dtype=np.float32)
 
 roi = np.array([], dtype=np.float32)
-scales = np.array([], dtype=np.float32)
```

Comment:
should we add a test where scales exists and sizes does not ?

Reply:
Thanks. There are already such tests before this PR submitted (e.g., https://github.com/onnx/onnx/blob/master/onnx/backend/test/case/node/resize.py#L341) .